src/xattrs.c

```diff
@@ -37,6 +37,14 @@ enum xattrs_action_type_t_ {
 
 typedef enum xattrs_action_type_t_ xattrs_action_type_t;
 
+/* If SWUPD_WITH_XATTRS is not defined, create functions to override
+ * glibc's interface to the system calls
+ */
+#ifndef SWUPD_WITH_XATTRS
+#define lgetxattr(p,n,b,l) (-1)
+#define llistxattr(p,b,l) (0)
```

Comment:
Thanks for clarifying. I think returning `0` would make more sense in this context. But now I'm wondering if `lgetxattr` should also return `0`? Unless zero-length xattr values ever make sense?